lib/runner.js

```diff
@@ -265,10 +265,17 @@ Runner.prototype.fail = function(test, err) {
  * @param {Error} err
  */
 Runner.prototype.failHook = function(hook, err) {
+  hook.originalTitle = hook.originalTitle || hook.title;
   if (hook.ctx && hook.ctx.currentTest) {
-    hook.originalTitle = hook.originalTitle || hook.title;
     hook.title =
       hook.originalTitle + ' for "' + hook.ctx.currentTest.title + '"';
+  } else {
+    var parent = hook.parent.title
```

Comment:
`parentTitle`?

Reply:
`hook.parent.title` is correct, it's the suite title.

Reply:
Was in reference to `var parent`